### PR TITLE
Add Commercio.network

### PR DESCRIPTION
**Pull request by zenit70**
Add Commercio.network chain.

Thanks

### CODE REVIEW

File: commercionetwork/chain.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "commercionetwork",
+  "chain_id": "commercio-3",
+  "pretty_name": "Commercio.network",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "did:com:",
```
Comment (sunnya97) [inline commercionetwork/chain.json:8]: Is this a valid bech32 path?

Comment (JeremyParish69) [reply commercionetwork/chain.json:8]: it's been registered to SLIP173 a while

Comment (zenit70) [reply commercionetwork/chain.json:8]: @sunnya97 Yes, is a valid path. As specified in https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki#Bech32 the range of characters allowed for hrp should also include “:”

Comment (sunnya97) [reply commercionetwork/chain.json:8]: Wow, TIL

**Pull request merged**